pages/api/collect.ts

```diff
@@ -88,7 +104,21 @@ export default async (req: NextApiRequestCollect, res: NextApiResponse) => {
 
   await useSession(req, res);
 
-  const session = req.session;
+  const { session, website } = req.session;
+
+  // Check collection limit
+  if (process.env.ENABLE_COLLECT_LIMIT) {
+    const userId = website.userId ? website.userId : website.team.userId;
```

Comment:
The team user id does not mean they are the owner. The role defines the owner.

Reply:
> A majority of this logic should not be in the app. Only we care about limits, the end user never will. The app should only care if a number is hit, then abort. The real logic will either live on cloud side or in cron scripts that populate redis caches, etc.

Right, I imagine cloud, will populate the 'user max' number through redis, but those membership total mechanisms isn't built yet.  I can make assumptions for it now though.

Reply:
We should get rid of user_id on the TEAM table if it doesn't represent owner anymore.